consensus/parlia/parlia.go

```diff
@@ -75,9 +75,9 @@ var (
 	diffInTurn = big.NewInt(2)            // Block difficulty for in-turn signatures
 	diffNoTurn = big.NewInt(1)            // Block difficulty for out-of-turn signatures
 	// 100 native token
-	maxSystemBalance                 = new(big.Int).Mul(big.NewInt(100), big.NewInt(params.Ether))
-	verifyVoteAttestationFailedGauge = metrics.NewRegisteredGauge("parlia/verifyVoteAttestationFailed", nil)
-	updateAttestationFailedGauge     = metrics.NewRegisteredGauge("parlia/updateAttestationFailed", nil)
+	maxSystemBalance           = new(big.Int).Mul(big.NewInt(100), big.NewInt(params.Ether))
+	verifyVoteAttestationError = metrics.NewRegisteredGauge("parlia/verifyVoteAttestation/error", nil)
+	updateAttestationError     = metrics.NewRegisteredGauge("parlia/updateAttestation/error", nil)
```

Comment:
better to use `NewRegisteredCounter` as it is a counter?

Reply:
accept